be/src/vec/columns/column_nullable.h

```diff
@@ -353,23 +349,41 @@ class ColumnNullable final : public COWHelper<IColumn, ColumnNullable> {
 
         if (!nullable_rhs.is_null_at(row)) {
             nested_column->replace_column_data(*nullable_rhs.nested_column, row, self_row);
-        } else {
-            nested_column->replace_column_data_default(self_row);
         }
     }
 
-    void replace_column_data_default(size_t self_row = 0) override {
-        LOG(FATAL) << "should not call the method in column nullable";
-        __builtin_unreachable();
-    }
-
     MutableColumnPtr convert_to_predicate_column_if_dictionary() override {
         nested_column = get_nested_column().convert_to_predicate_column_if_dictionary();
         return get_ptr();
     }
 
     double get_ratio_of_default_rows(double sample_ratio) const override {
```

Comment:
should we really need the func? seems unless in doris code

Reply:
used on ColumnObject